main.css

```diff
@@ -8,8 +8,10 @@ body {
   left: -17px;
   background: #fff;
   position: relative;
-  width: 375px;
+  width: 100vw;
+  max-width: 375px;
   margin: auto;
+  overflow: hidden;
```

Comment:
- [ ] Thank you for making an effort here although I think there are still places in the CSS that is preventing the site from being flexible. As the previous reviewer suggested please make sure that the design is flexible. avoid setting constant widths in your CSS. look around the CSS to find where you might have set constant widths 

![image](https://user-images.githubusercontent.com/18730106/153495573-1dceeec6-f957-4064-93f4-73da1981ef98.png)